shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -222,7 +222,9 @@ void PostAccessibilityNotification(UIAccessibilityNotifications notification,
     if (last_focused_semantics_object_id_ != kSemanticObjectIdInvalid) {
       // Tries to refocus the previous focused semantics object to avoid random jumps.
       nextToFocus = [objects_.get() objectForKey:@(last_focused_semantics_object_id_)];
-      if (!nextToFocus && root) {
+      if (nextToFocus) {
+        nextToFocus = FindFirstFocusable(nextToFocus);
+      } else if (root) {
```

Comment:
Since you do these 2 checks I think it would be helpful if you pulled this out into a function like:
```c++
Object* FindFirstFocusable(Object* one, Object* two) {
  if (one) {
    return FindFirstFocusable(one);
  } else if (two) {
    return FindFirstFocusable(two);
  }
  return nullptr;
}
```

This function is a monster, the more duplicate logic we can pull out the better imo.

Logic seems find though.